tests/taskcluster/tasks/marionette_js_tv_tests.yml

```diff
@@ -0,0 +1,45 @@
+# Chunking task for marionette js tests...
+---
+reruns: 4
+task:
+  metadata:
+    name: 'Marionette JS tests for tv'
+    description: Run entire suite of marionette js tests for tv
+
+  workerType: '{{workerType}}'
+  provisionerId: '{{provisionerId}}'
+  scopes:
+    # Source caches contains various sources (including a tarball of gaia source
+    # tree)
+    - docker-worker:cache:gaia-misc-caches
+    - docker-worker:image:quay.io/mozilla/gaia-taskenv:*
+
+  payload:
+    image: quay.io/mozilla/gaia-taskenv:0.8.9
```

Comment:
You'll be able to track the work here: https://bugzilla.mozilla.org/show_bug.cgi?id=1165488
If it lands before that bug is resolved, you have nothing to worry about. I'll update it. :)